cogito/ghcommitsink_test.go

```diff
@@ -23,17 +23,21 @@ func TestSinkGitHubCommitStatusSendSuccess(t *testing.T) {
 	var ghReq github.AddRequest
 	var URL *url.URL
 	ts := testhelp.SpyHttpServer(&ghReq, nil, &URL, http.StatusCreated)
+	gitHubSpyURL, err := url.Parse(ts.URL)
+	if err != nil {
+		t.Fatalf("error parsing SpyHttpServer URL: %s", err)
```

Comment:
same comment about `assert.`

Reply:
Fixed in: https://github.com/Pix4D/cogito/commit/66f1cf52187b4780f4fb0775009bab43f2a2155d